src/commentsAPI.js

```diff
@@ -0,0 +1,28 @@
+class commentsApi {
+    static fetchRecipes = (item, appID) => new Promise((resolve, reject) => {
+      if (resolve) {
+        return resolve(
+          fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appID}/comments?item_id=${item}`)
+            .then((res) => res.json())
+            .then((json) => json),
+
+        );
+      } return reject();
+    });
+
+    static postComment = (datos, appID) => new Promise((resolve, reject) => {
+      if (resolve) {
+        return resolve(
+          fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appID}/comments`, {
+            method: 'POST',
+            body: JSON.stringify(datos),
+            headers: {
+              'Content-type': 'application/json; charset=UTF-8',
+            },
+          }),
+        );
+      } return reject();
+    });
```

Comment:
- [ ]  It will be nice to use  `async` and `await` with the `fetch` method to send and receive data.